src/routes/user/user.service.ts

```diff
@@ -126,3 +128,27 @@ export const verifyEmail = async (token: string, email: string): Promise<any> =>
 
   return Format.badRequest('Invalid verification link');
 };
+
+/**
+ * Refresh Access Token
+ *
+ * @param {user} user - user
+ */
+export const refreshAccessToken = async (user: UserDoc | undefined): Promise<any> => {
+  logger.log(`[${SERVICES_NAMES.refreshAccessToken}] is called`);
+
+  if (!user) {
+    return Format.notFound('User not found');
+  }
```

Comment:
We are already handling this case in the middleware, but since we're passing it from the middleware to the controller, we get this case of undefined. While doing req.user, we can do `const user = req.user as UserDoc` to explicitly tell typescript that we'll make sure it doesn't come as undefined. We can handle this case in controller.